invenio_files_rest/storage.py

```diff
@@ -28,9 +28,14 @@
 
 import hashlib
 import time
+import tempfile
```

Comment:
But I don't see how the storage factory will be built to support contrib. Can you enlight me ?